src/client.ts

```diff
@@ -38,7 +38,7 @@ export class Client {
   public url?: string;
   /** `name` is a human readable string associated with this app on the Lattice */
   private name: string;
-  private key: KeyPair;
+  public key: KeyPair;
```

Comment:
Uhhh I'm not sure we want this. Doing so will expose the private key too

Reply:
I've always been confused by access modifiers in TypeScript.

Changing this to `public` so it can be referenced elsewhere in the code make sense; however, keeping it `private` for the purposes of keeping it "_safe_" seems inconsequential, no?

Looking for clarification: @douglance.

<hr/>
<img width="684" alt="Screenshot 2023-01-17 at 11 46 06 AM" src="https://user-images.githubusercontent.com/276137/212960087-7110f39b-3bb4-417e-96da-82ba703a0636.png">

[StackOverflow Source](https://stackoverflow.com/a/68289451)

Reply:
It was changed because we access that value elsewhere in the codebase. I reverted that change and added `ts-expect-error` to ignore accessing the private value.

`public` and `private` are TypeScript-only modifiers. They aren't preserved _at all_ when we compile to JS. 

There _is_ a way to make JS values more [private](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields). The data can still be extracted out of those values if an attacker wanted them.